read-fonts/src/tables/dsig.rs

```diff
@@ -0,0 +1,58 @@
+//! The [DSIG](https://learn.microsoft.com/en-us/typography/opentype/spec/dsig) table
+
+include!("../../generated/generated_dsig.rs");
+
+#[cfg(test)]
+mod tests {
+    use font_test_data::bebuffer::BeBuffer;
+
+    use super::{Dsig, PermissionFlags};
+    use crate::{FontData, FontRead};
+
+    /// An empty, dummy DSIG, as inserted by fonttools.
```

Comment:
Maybe you can add a `Dsig::DUMMY` variable? I suppose that most workflows dealing with DSIG are going to use it.

Reply:
Was tempted but it's less of a special case than I'd worried before looking under the hood; it's just a regular table construction with 0 signature records (as seen in `write-fonts` where we construct it in a type-safe way), and so I don't think we need to hang onto the raw bytes anywhere in the end :+1: